src/elements/content-sidebar/__tests__/ContentSidebar.test.js

```diff
@@ -76,6 +76,21 @@ describe('elements/content-sidebar/ContentSidebar', () => {
             expect(instance.fetchFile).not.toBeCalled();
             expect(instance.setState).not.toBeCalled();
         });
+
+        test('should not fetch the file data if the id has not changed', () => {
+            const wrapper = getWrapper({ fileId: '123' });
+            const instance = wrapper.instance();
+            const newProps = { fileId: '123' };
+
+            instance.fetchFile = jest.fn();
+            instance.setState({ view: 'activityFeed' });
+            instance.setState = jest.fn();
+
+            instance.componentDidUpdate(newProps);
+
+            expect(instance.fetchFile).not.toBeCalled();
+            expect(instance.setState).not.toBeCalled();
+        });
```

Comment:
Isn't that an exact copy of the above test?

Reply:
Unresolving this comment as I don't understand why do we need this duplicated test?